internal/mempool/reactor.go

```diff
@@ -153,6 +153,15 @@ func (r *Reactor) handleMempoolMessage(ctx context.Context, envelope *p2p.Envelo
 					// problem.
 					continue
 				}
+				if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
+					// this shouldn't be an error
+					// that gets logged, or
+					// propagated up, but the
+					// calling loop will abort the
+					// iterator.
```

Comment:
```suggestion
					// I have eaten the error
					// that was in the log
					// and that you were propagating
					// up to the caller
					// I'm sorry
					// the calling loop was delicious
					// so aborted
					// and so iterator.
```

Reply:
I didn't mean for you to actually apply that haha.

Reply:
lol, I have fixed it.